### ISSUE TITLE
[BUG]  Labels must be unique

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5532]
This is the exact same bug as https://github.com/rokwire/illinois-app/issues/4874

 To be clear this is a bug that is going to affect more than this data usage statement. Anywhere in the code where a button label or link ID is repeated. Anywhere that just reads "Button double tap to activate" like the toggle for Directory Visibility. I cannot open a bug for every instance. This will undoubtedly need developers to CTRL + F in the code to find the reoccurrences and instances of these. I'm also happy to do a screen share if that helps.

Per @TodorBachvarov: 
"Hi @acovher2, the described issues should be fixed in version 7.3.22. If you find any new additional issues, describe them in new tickets please."

This ticket is representing WEB app version 7.3.24 in which I'm blocked from retesting due to Sign on with NetID not working. 

![Image](https://github.com/user-attachments/assets/ebde14a2-4ed8-46b2-b2dd-bee53cf25af0)

![Image](https://github.com/user-attachments/assets/a3fb9a46-a765-48ce-8ff0-ce44dc0e5e37)

![Image](https://github.com/user-attachments/assets/82731aab-8bb0-49d0-b78d-47328c49dd3b)

**Comment by acovher2** [comment]
![Image](https://github.com/user-attachments/assets/479864d8-5fb9-488c-90ea-79718338c04d)

@TodorBachvarov @pmarkhennessy To clarify, will the google Flutter issues prevent things like this being fixed on the App, or only the web?  There are going to be items like this Delete button in the App that need addressed and I'm just trying to understand fully if the limitation only affects Web or if it affects Both Web and App. Please let me know.
